CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 Please use this place to add information of work that hasn't been released,
 and specify if that is backwards-compatible.
 
+# 31.2.0
+
+* Add a `area_for_code_type` method to the MapIt API adapter.
```

Comment:
It's called [`area_for_code`](https://github.com/alphagov/gds-api-adapters/blob/master/lib/gds_api/mapit.rb#L15) now.